regression/strings-smoke-tests/java_append_object/test.desc

```diff
@@ -1,8 +1,9 @@
 FUTURE
 test_append_object.class
---refine-strings --string-max-length 1000
+--refine-strings --verbosity 10 --string-max-length 1000
 ^EXIT=0$
 ^SIGNAL=0$
 ^VERIFICATION SUCCESSFUL$
 --
+non equal types
 Issue: diffblue/test-gen#82
```

Comment:
There is a problem in the original test description file: this comment "Issue: ..." should have a preceding `--` line to separate strings-not-to-be-found-in-output from comments.